uint8 command
uint8 status
float32 vel_x
float32 vel_y
float32 vel_th

